src/Symfony/Component/DomCrawler/Tests/CrawlerTest.php

```diff
@@ -64,13 +64,12 @@ public function testAddInvalidType()
     }
 
     /**
-     * @expectedException \InvalidArgumentException
-     * @expectedExceptionMessage Nodes set in a Crawler must be DOMElement or DOMDocument instances, "DOMNode" given.
+     * @group legacy
      */
-    public function testAddInvalidNode()
+    public function testAddMultipleDocumentNode()
     {
-        $crawler = new Crawler();
-        $crawler->add(new \DOMNode());
+        $crawler = $this->createTestCrawler();
+        $crawler->addHtmlContent('<html><div class="foo"></html>', 'UTF-8');
```

Comment:
I wanted to test the deprecation notice, but it's out of scope of this PR, so just removed it.